src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -78,7 +78,18 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
   userState = container.resolve(UserState),
   teamState = container.resolve(TeamState),
 }) => {
-  const defaultProtocol = teamState.teamFeatures().mls?.config.defaultProtocol;
+  const {isTeam, isMLSEnabled: isMLSEnabledForTeamUser} = useKoSubscribableChildren(teamState, [
+    'isTeam',
+    'isMLSEnabled',
+  ]);
+
+  const enableMlsToggle = isMLSEnabledForTeamUser && Config.getConfig().FEATURE.ENABLE_MLS;
```

Comment:
```suggestion
  const enableMLSToggle = isMLSEnabledForTeamUser && Config.getConfig().FEATURE.ENABLE_MLS;
```
I'm straight up copying Przemek reviews